samples/matter/lock/Kconfig

```diff
@@ -60,6 +60,12 @@ config APPLICATION_OTHER_LABEL
 
 endif
 
+config THREAD_WIFI_SWITCHING_CLI_SUPPORT
```

Comment:
How about moving it few lines higher to be included in the if THREAD_WIFI_SWITCHING condition?